app/models/thredded/topic.rb

```diff
@@ -44,7 +44,7 @@ class Topic < ActiveRecord::Base
                inverse_of:    :topics,
                counter_cache: :topics_count
 
-    has_many :posts,
+    has_many :posts, -> { order_oldest_first },
```

Comment:
@glebm Ugh, you're right -- `order_newest_first` becomes very complicated. if we redefine it as reorder() then it starts a pattern of unexpected behaviour.

Ok, I guess so. I notice about 5 places where I'd like to see an order and it's missing. Only one of them is critical (two are in specs, and the other two are only used when there is only one post, so I've just added a comment)

https://github.com/thredded/thredded/blob/master/app/commands/thredded/mark_all_read.rb#L9. That this is in very recently added code does suggest to me that order_oldest_first is a kind of natural order that ought to be default, but I agree the problems around defaults in scopes and associations are too big. Just have to watch out for this in PRs.

Have also spotted a marginal error in a spec/features/page_object. Really don't like these. Just saying. They make the feature code harder to reason about and harder to write.